packages/@okta/vuepress-site/docs/reference/api/authorization-servers/index.md

```diff
@@ -161,6 +161,8 @@ When you use these API endpoints to create or modify an Authorization Server res
 
 * If set to `CUSTOM_URL`, then in responses, `issuer` is the custom domain URL configured in the administration user interface.
 
+* If set to `DYNAMIC`, then in responses, `issuer` is the custom domain URL if the token was issued by custom domain or the Okta org's original domain URL if the token was issued by the original Okta org domain.
```

Comment:
`issuer` is the custom domain URL if the oauth2 request was sent to the custom domain or the Okta domain URL if the oauth2 request was sent to the original Okta org domain.

Reply:
updated